client/src/app/core/ui-services/base-sort-list.service.ts

```diff
@@ -218,59 +218,72 @@ export abstract class BaseSortListService<V extends BaseViewModel> {
     protected updateSortedData(): void {
         if (this.inputData) {
             const property = this.sortProperty as string;
-            const intl = new Intl.Collator(this.translate.currentLang);
-            this.outputSubject.next(
-                this.inputData.sort((itemA, itemB) => {
-                    const firstProperty = this.ascending ? itemA[property] : itemB[property];
-                    const secondProperty = this.ascending ? itemB[property] : itemA[property];
-                    if (typeof firstProperty !== typeof secondProperty) {
-                        // undefined/null items should always land at the end
-                        if (!firstProperty) {
-                            return 1;
-                        } else if (!secondProperty) {
-                            return -1;
-                        } else {
-                            throw new TypeError('sorting of items failed because of mismatched types');
-                        }
+
+            // filter *false-like* string objects. They can not reliable be sorted.
+            const positives = [];
+            const negatives = [];
+            for (const item of this.inputData) {
```

Comment:
@FinnStutzenstein I'd like to have your opinion on this, since I feel that a pretty overkill. Yet somehow, handling all the neglection of empty string parameters inside the sorting function can always lead to unwanted results.